src/scripts/modal.js

```diff
@@ -0,0 +1,23 @@
+function modal() {
+  const heroContent = document.querySelector(".hero__content");
+  const modalContent = document.querySelector("#modal");
+  const checkOfferBtn = document.querySelector(".hero--cta");
+  const confrimModalBtn = document.querySelector(".modal-btn--confirm");
+  const closeModalBtn = document.querySelector(".modal-btn--close");
+  const closeModalIcon = document.querySelector(".modal-icon--close");
+
+  function closeModal() {
```

Comment:
This function is defined inside `modal` fn. Let's call it `close`

Reply:
Changed.